### ISSUE TITLE
training script for your results

### ISSUE THREAD

**Issue by ryanwebster90** [rosinality/stylegan2-pytorch#134]
Hello,
I'm having a bit of trouble reproducing your results on FFHQ. After 800k iterations I get an FID of 8 (official FID, the pytorch FID is still not indicative of quality), and you can really tell that the quality is poor. Especially on harder datasets such as LSUN-Cat. I'm currently using 1 1080 Ti with a batch size of 4. 

Could you provide your training configuration when you achieved FID of 4.5 (the weights in 550k FFHQ model). Perhaps the batch size is the source of the lower quality.

**Comment by rosinality** [comment]
I have trained model with total batch sizes 32. I think it will be quite hard to train with batch size 4.

**Comment by ryanwebster90** [comment]
Well it works surpisingly well, just slightly worse than the model you've provided, but on harder datasets like LSUN cat the drop in quality is pretty noticable. for a 1080 with 11 gb gpu ram, a batchsize of 32 would be 8 gpus without gradient accumulation.
